website/static/js/script.js

```diff
@@ -42,31 +46,34 @@
     }
   })
 
-  supporters.then(function (entries) {
+  supporters.then(function (orders) {
     var fragment = document.createDocumentFragment();
     var nodes = [];
-    var supportersToDisplay = entries.slice(0, 10);
+    var uniqueSupporters = uniqify(orders.nodes.sort((a, b) => a.totalDonations.value < b.totalDonations.value), o => o.fromAccount.slug);
+    var supportersToDisplay = uniqueSupporters.slice(0, 10);
 
     supportersToDisplay.forEach(function(supporter) {
+      var { fromAccount: { name, slug, website, twitterHandle, imageUrl }, totalDonations } = supporter;
+
       var img = new Image();
-      img.src = supporter.avatar;
+      img.src = imageUrl;
       img.width = 80;
       img.onload = function(e) {
         e.currentTarget.parentNode.classList.remove('supporter--skeleton');
       }
+      img.alt = name;
 
       var link = document.createElement('a');
       link.className = 'supporter supporter--skeleton';
       link.rel = 'nofollow';
-      if (supporter.website) {
-        link.href = supporter.website;
-        img.alt = supporter.website;
-      } else if (supporter.twitterHandle) {
-        link.href = 'https://twitter.com/' + supporter.twitterHandle
-        img.alt = supporter.twitterHandle;
+      link.title = `${name} with $${totalDonations.value} total`;
```

Comment:
I don't see much value in adding this title text. Mobile and keyboard-focused users won't see it, and screen readers will be preferring the alt text already provided.

Reply:
I fell it was a nice touch for desktop browsers, you get a tooltip when hovering the image, answering "who's that? how much they donated?".

No issue removing it, this is not my website :-)

Reply:
I think this is pretty cool for desktop. Only 4% of users are on mobile.